src/main/java/com/forgeessentials/scripting/ScriptMethods.java

```diff
@@ -502,7 +502,233 @@ public String getHelp()
             return "`random <success-chance-in-percent>`  \nThis method will randomly success or fail.  \nIf it fails, the rest of the script will not be executed any more.";
         }
     };
+	
+	public static final ScriptMethod expCheck = new ScriptMethod() {
+        @Override
+        public boolean process(final ICommandSender sender, String[] args)
+        {
+            if (args.length >= 1)
+			{
+				if (!(sender instanceof EntityPlayerMP))
+					throw new MissingPlayerException();
+				
+				int experienceLevel = Integer.parseInt(args[1]);
+				String operator = args.length >= 2 ? args[2] : "EQ";
+				if (operator.equals("EQ"))
+				{
+					return ((EntityPlayerMP) sender).experienceLevel == experienceLevel;
+				}
+				else if (operator.equals("LT"))
+				{
+					return ((EntityPlayerMP) sender).experienceLevel < experienceLevel;
+				}
+				else if (operator.equals("LTE"))
+				{
+					return ((EntityPlayerMP) sender).experienceLevel <= experienceLevel;
+				}
+				else if (operator.equals("GT"))
+				{
+					return ((EntityPlayerMP) sender).experienceLevel > experienceLevel;
+				}
+				else if (operator.equals("GTE"))
+				{
+					return ((EntityPlayerMP) sender).experienceLevel >= experienceLevel;
+				}
+				else
+					return false;
+			}
+			else
+				return false;
+        }
+
+        @Override
+        public String getHelp()
+        {
+            return "`expCheck <expLevel> [operator -- LT, LTE, GT, GTE, or EQ]`  \nThis method will check the expLevel of a player and compare it to a given value. If no operator is specified, EQ will be used as default";
```

Comment:
People are idiots, what do the operators mean?

Or you can just use `<`, `=<`, `>`, `>=`, and `=`. Not sure how that would work out though.